keyboards/jaykeeb/aumz_work/hotswap/info.json

```
@@ -0,0 +1,257 @@
+                {"label": "\\", "matrix": [2, 13], "x": 13.5, "y": 1, "w": 1.5},
+                {"label": "End", "matrix": [1, 14], "x": 15, "y": 1},
+
+                {"label": "Caps Lock", "matrix": [2, 0], "x": 0, "y": 2, "w": 1.75},
+                {"label": "A", "matrix": [2, 1], "x": 1.75, "y": 2},
+                {"label": "S", "matrix": [2, 2], "x": 2.75, "y": 2},
+                {"label": "D", "matrix": [2, 3], "x": 3.75, "y": 2},
+                {"label": "F", "matrix": [2, 4], "x": 4.75, "y": 2},
+                {"label": "G", "matrix": [2, 5], "x": 5.75, "y": 2},
+                {"label": "H", "matrix": [2, 6], "x": 6.75, "y": 2},
+                {"label": "J", "matrix": [2, 7], "x": 7.75, "y": 2},
+                {"label": "K", "matrix": [2, 8], "x": 8.75, "y": 2},
+                {"label": "L", "matrix": [2, 9], "x": 9.75, "y": 2},
+                {"label": ";", "matrix": [2, 10], "x": 10.75, "y": 2},
+                {"label": "'", "matrix": [2, 11], "x": 11.75, "y": 2},
+                {"label": "Enter", "matrix": [2, 12], "x": 12.75, "y": 2, "w": 2.25},
+                {"label": "Page Up", "matrix": [2, 14], "x": 15, "y": 2},
+
+                {"label": "Shift", "matrix": [3, 0], "x": 0, "y": 3, "w": 2.25},
+                {"label": "Z", "matrix": [3, 2], "x": 2.25, "y": 3},
+                {"label": "X", "matrix": [3, 3], "x": 3.25, "y": 3},
+                {"label": "C", "matrix": [3, 4], "x": 4.25, "y": 3},
+                {"label": "V", "matrix": [3, 5], "x": 5.25, "y": 3},
+                {"label": "B", "matrix": [3, 6], "x": 6.25, "y": 3},
+                {"label": "N", "matrix": [3, 7], "x": 7.25, "y": 3},
+                {"label": "M", "matrix": [3, 8], "x": 8.25, "y": 3},
+                {"label": ",", "matrix": [3, 9], "x": 9.25, "y": 3},
+                {"label": ".", "matrix": [3, 10], "x": 10.25, "y": 3},
+                {"label": "/", "matrix": [3, 11], "x": 11.25, "y": 3},
+                {"label": "Shift", "matrix": [3, 12], "x": 12.25, "y": 3, "w": 1.75},
+                {"label": "\u2191", "matrix": [3, 13], "x": 14, "y": 3},
+                {"label": "Page Down", "matrix": [3, 14], "x": 15, "y": 3},
+
+                {"label": "Ctrl", "matrix": [4, 0], "x": 0, "y": 4, "w": 1.25},
+                {"label": "GUI", "matrix": [4, 1], "x": 1.25, "y": 4, "w": 1.25},
+                {"label": "Alt", "matrix": [4, 2], "x": 2.5, "y": 4, "w": 1.25},
+                {"label": "Space", "matrix": [4, 6], "x": 3.75, "y": 4, "w": 6.25},
+                {"label": "Alt", "matrix": [4, 10], "x": 10, "y": 4, "w": 1.25},
+                {"label": "Ctrl", "matrix": [4, 11], "x": 11.25, "y": 4, "w": 1.25},
+                {"label": "\u2190", "matrix": [4, 12], "x": 13, "y": 4},
+                {"label": "\u2193", "matrix": [4, 13], "x": 14, "y": 4},
+                {"label": "\u2192", "matrix": [4, 14], "x": 15, "y": 4}
+            ]
+        },
```

matching layouts condensed into one, `"layout_aliases"` used to conform to PR Checklist regarding more than one physical layout.
```suggestion
    },
    "layout_aliases": {
        "LAYOUT_all": "LAYOUT_65_ansi_blocker"
    },
    "community_layouts": ["65_ansi_blocker", "65_ansi_blocker_tsangan"],
    "layouts": {
```

---

oke done @dunk2k :)